.teamcity/generated/ImageValidation.kts

```diff
@@ -25,9 +25,9 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
 
 object image_validation: BuildType({
 	 name = "Validation of Size Regression - Staging Docker Images (Windows / Linux)"
-	 buildNumberPattern="%dockerImage.teamcity.buildNumber%-%build.counter%"
+	 buildNumberPattern="validate-%dockerImage.teamcity.buildNumber%-%build.counter%"
```

Comment:
I'd expected that a build number in this configuration would have the same format as other docker-related configurations, namely "%dockerImage.teamcity.buildNumber%-%build.counter%". The "validate-" prefix just duplicates the information from the build configuration name.